tests/sanity/ignore-2.10.txt

```diff
@@ -462,5 +462,16 @@ plugins/modules/system/xfconf.py validate-modules:parameter-state-invalid-choice
 plugins/modules/system/xfconf.py validate-modules:return-syntax-error
 plugins/modules/web_infrastructure/jenkins_plugin.py use-argspec-type-path
 plugins/modules/web_infrastructure/rundeck_acl_policy.py pylint:blacklisted-name
+tests/integration/targets/django_manage/files/base_test/1045-single-app-project/single_app_project/manage.py metaclass-boilerplate      # django generated code
+tests/integration/targets/django_manage/files/base_test/1045-single-app-project/single_app_project/manage.py future-import-boilerplate  # django generated code
+tests/integration/targets/django_manage/files/base_test/1045-single-app-project/single_app_project/manage.py pep8:E302  # django generated code
+tests/integration/targets/django_manage/files/base_test/1045-single-app-project/single_app_project/manage.py pep8:E305  # django generated code
+tests/integration/targets/django_manage/files/base_test/1045-single-app-project/single_app_project/manage.py shebang    # django generated code
+tests/integration/targets/django_manage/files/base_test/simple_project/p1/manage.py compile-2.6   # django generated code
+tests/integration/targets/django_manage/files/base_test/simple_project/p1/manage.py compile-2.7   # django generated code
+tests/integration/targets/django_manage/files/base_test/simple_project/p1/manage.py metaclass-boilerplate       # django generated code
+tests/integration/targets/django_manage/files/base_test/simple_project/p1/manage.py future-import-boilerplate   # django generated code
+tests/integration/targets/django_manage/files/base_test/simple_project/p1/p1/settings.py metaclass-boilerplate       # django generated code
+tests/integration/targets/django_manage/files/base_test/simple_project/p1/p1/settings.py future-import-boilerplate   # django generated code
```

Comment:
So much for shrinking these files ;-)

Reply:
Tell me about it... I was heart broken to do that, but this is exactly the type of case where this actually makes sense. 😃

Reply:
As soon as that nios PR comes through, I will redeem myself :)

Reply:
Out of curiosity, even though these files are auto-generated, does Django complain if we fix all (or at least some) of the sanity errors found in them? If the tests still work fine with the fixes, we could save some lines in the ignore.txt files :)

Reply:
TBH I haven't tried that, exactly because the reality of the projects out there will not conform to the sanity tests we have here. It kind of defeats the purpose of the tests, IMO.

Reply:
Well, the reality of the projects are not part of this collection, so they don't have to satisfy our sanity checks :) Most of the changes needed to reduce ignore.txt lines should not have any impact on the tests.